lib/fedex_rest_api/track.rb

```diff
@@ -3,23 +3,24 @@
 require "fedex_rest_api/base"
 
 class FedexRestApi::Track
-  attr_reader :tracking_object
+  attr_reader :tracking_object, :environment
 
   TRACK_URL = "/track/v1/trackingnumbers"
 
-  def initialize(tracking_object)
+  def initialize(tracking_object, environment: nil)
```

Comment:
Breaking out environment is a nice touch 💯 I wonder if we should default it to something like `Base::SANDBOX` just for extra clarity?

Reply:
Can do!